distribution/docs/src/main/resources/_contents/_introduction/introduction-contents.adoc

```diff
@@ -1,11 +1,13 @@
 
-Codice ${branding} is an open source, modular integration framework building on ${ddf-branding} with the addition of DoD/IC and NATO support capabilities.
+Codice ${cal-branding} is an open source, modular integration framework building on ${ddf-branding} with the addition of DoD/IC and NATO support capabilities.
 
-=== ${branding} Features
+==== ${cal-branding} Features
```

Comment:
`<cal-branding>Alliance</cal-branding>` from the top-level pom file: 

https://github.com/codice/alliance/blob/master/pom.xml#L83